spec/template.dd

```diff
@@ -528,50 +528,50 @@ $(GNAME TemplateAliasParameterDefault):
 )
 
     $(P Alias parameters enable templates to be parameterized with
-        any type of D symbol, including global names, local names,
+        any type of D symbol, including type names, global names, local names,
```

Comment:
We should make it clear that built-in types can't be passed to an alias parameter.